javascript-light-bulb/main.js

```diff
@@ -1,15 +1,16 @@
 var $lightBulb = document.querySelector('.bulb');
 var $lightStatus = document.querySelector('.off');
-var clicks = 0;
+var clicks = true;
```

Comment:
You should revise the name of this variable to be accurate. Maybe something like `isLightOn`